ThreeBotPackages/threefold/tfgrid_directory/package.py

```diff
@@ -18,9 +23,21 @@ def start(self):
         :return:
         """
         self.bcdb.models_add(path=self.package_root + "/models")
+        # start one worker to sync farms
+        j.servers.myjobs.workers_tmux_start(1)
+        gevent.spawn_later(DIR_SYNC_TIME, self.sync_directory)
```

Comment:
I don't think we should delay the execution of the first run that much. What's the reason for this ?